lib/main.dart

```diff
@@ -21,6 +23,14 @@ void main() async {
   WidgetsFlutterBinding.ensureInitialized();
   MediaKit.ensureInitialized();
   GoRouter.optionURLReflectsImperativeAPIs = true;
+  if (PlatformInfos.isLinux) {
+    Hive.init((await getApplicationSupportDirectory()).path);
+  } else {
+    await Hive.initFlutter();
+  }
+
+  GetItInitializer().setUp();
```

Comment:
you change it to Future, so this should be `await`

Reply:
it's void check it again

Reply:
?